chainer/functions/math/inv.py

```diff
@@ -0,0 +1,101 @@
+import six
+import numpy.linalg
+
+import cupy
```

Comment:
Please remove this line. Chainer must not directly import the cupy. Use the `chainer.cuda.cupy`.
For example,
https://github.com/pfnet/chainer/blob/83800a77b9118fa4d7e98b03056a95f488e96a51/chainer/functions/array/select_item.py#L49